doc/api/deprecations.md

```diff
@@ -534,6 +534,21 @@ deprecated. Please use `ServerResponse.prototype.writeHead()` instead.
 *Note*: The `ServerResponse.prototype.writeHeader()` method was never documented
 as an officially supported API.
 
+<a id="DEP0064"></a>
+### DEP0064: outgoingMessage.\_headers, outgoingMessage.\_headerNames
```

Comment:
Note: depending on when this lands, the code may need to be updated (e.g. if any other deprecations land first) ... I don't suspect that will be an issue, but just to be aware